pkgs/data/themes/catppuccin-sddm-corners/default.nix

```diff
@@ -14,8 +15,14 @@ stdenvNoCC.mkDerivation {
     hash = "sha256-sTnt8RarNXz3RmYfmx4rD+nMlY8rr2n0EN3ntPzOurw=";
   };
 
+  # Might also need qt-svg, qt-quickcontrols2
+  propagatedBuildInputs = [
+    plasma5Packages.qtgraphicaleffects
+  ];
+
   dontConfigure = true;
-  dontBuild = true;
+  #dontBuild = true;
```

Comment:
assuming the postFixup doesn't trigger with dontBuild = true? if so.. remove dont comment